integrations/twilio/client.go

```diff
@@ -40,6 +44,49 @@ func New(vars sdkservices.Vars) sdkservices.Integration {
 				sdkmodule.WithArgs("to", "from_number?", "messaging_service_sid?", "body?", "media_url?", "content_sid?"),
 			),
 		),
+		connStatus(i),
 		sdkintegrations.WithConnectionConfigFromVars(vars),
 	)
 }
+
+// connStatus is an optional connection status check provided by the
+// integration with AutoKitteh. The possible results are "init required"
+// (indicating the connection is not yet usable), "using X" (indicating
+// one of multiple available authentication methods is in use), or
+// "initialized" when only one authentication method is available.
+func connStatus(i *integration) sdkintegrations.OptFn {
+	return sdkintegrations.WithConnectionStatus(func(ctx context.Context, cid sdktypes.ConnectionID) (sdktypes.Status, error) {
+		if !cid.IsValid() {
+			return sdktypes.NewStatus(sdktypes.StatusCodeWarning, "init required"), nil
+		}
+
+		vs, err := i.vars.Get(ctx, sdktypes.NewVarScopeID(cid))
+		if err != nil {
+			return sdktypes.InvalidStatus, err
+		}
+
+		var decodedVars webhooks.Vars
+		vs.Decode(&decodedVars)
+
+		s, err := sdktypes.ParseSymbol("AuthType")
+		if err != nil {
+			return sdktypes.InvalidStatus, err
+		}
+
+		at := vs.Get(s)
+		if !at.IsValid() || at.Value() == "" {
+			return sdktypes.NewStatus(sdktypes.StatusCodeWarning, "init required"), nil
+		}
+
+		// Align with:
+		// https://github.com/autokitteh/web-platform/blob/main/src/enums/connections/connectionTypes.enum.ts
```

Comment:
why ts and not the proto?

Reply:
AFAIK this is not related to any proto...?